JS/Tema 3 - Structura iterativa si listele/script-pretul-mediu.js

```diff
@@ -0,0 +1,16 @@
+console.log(`Tema 3`);
+console.log(`Cerinta 2`);
+
+//2. Urmarind coloana pret cumparare, creeaza o lista cu aceste date, si creeaza un algoritm ce iti 
+//afiseaza media preturilor de cumparare(pretul medi de cumparare al unei masini)
+
+const carBuyPrice = [16000, 9000, 17000, 20300, 6000, 8000, 10000, 6000, 35500, 18900, 41000, 22500, 20500, 11590];
+
+let carBuyPriceSum = 0;
+for (let i = 0; i < carBuyPrice.length; i++) {
+    carBuyPriceSum += carBuyPrice[i];
+}
+
+const avgCarBuyPrice = carBuyPriceSum / carBuyPrice.length;
+const avgCarBuyPriceRounded = Math.ceil(avgCarBuyPrice);
```

Comment:
Intereasanta abrodare, atata sa ai in vedere ca Math.ceil rotunjeste cu precadere la urmatoarea valoare, Math.floor rejunjeste spre valaorea precedenta, si Math.round ar fi cea mai neinfluentata rotunjire, respectiv cea mai corecta